src/app/GitUI/UserControls/FileStatusList.cs

```diff
@@ -375,6 +378,11 @@ private void SetFindInCommitFilesGitGrepVisibilityImpl(bool visible)
                 // Adjust sizes "automatically" changed by visibility
                 cboFindInCommitFilesGitGrep.Top = 0;
             }
+            else if (_formFindInCommitFilesGitGrep?.Visible is false or null && cboFindInCommitFilesGitGrep.Text.Length > 0)
```

Comment:
I prefer `is not true`

Reply:
I preferred that, too, until I trapped into
![image](https://github.com/user-attachments/assets/0dc4bd53-fb71-4c89-8f05-83a54486e880)

This made me think about replacing all `is not`.

Reply:
please explain what you mean...

Reply:
The change in the screenshot should have removed nullable only.
But I got confused by the inverse logic of "`is not`".
It would have been clearer if the old code had been `?.IsEmpty is true or null`.

I have changed the condition for this PR to match the recent code style as requested. We can finish the discussion later and, perhaps, adapt those 5 occurrences.

Reply:
There should probably be `is (false or null)` and `is not true` is shorter. But Russkie commented on it and has done that before.